### DISCUSSION TITLE
How to check usage

### DISCUSSION THREAD

**Discussion by Sy-Sang** [open-meteo/open-meteo#1039] [Q&A]
Is there an API to view the current monthly usage?

**Comment by patrick-zippenfenig** [comment]
Hi,
statistics are currently not exposed. We only collect statistics internally to identify users who greatly exceed the API limits. This is done by just parsing webserver logs + Metrics database. We would like to improve it, however it is currently not the highest priority.

**Comment by Sy-Sang** [comment]
thanks!
